csrc/scheduler/utils.cpp

```diff
@@ -558,10 +584,13 @@ PersistentBufferInfo persistentBuffers(Fusion* fusion) {
     // If the buffer doesn't depends on reduction tvs or there is a broadcasted
     // tv between reduction and persistent buffer. The re-calculation of the
     // persistent buffer doesn't need additional reductions and this buffer is
-    // projecatable.
+    // projecatable. When searching for broadcasted tvs, exclude the buffer
+    // itself.
     const auto& dep_vals = DependencyCheck::getAllValsBetween(
```

Comment:
nit: Move this into `getBufferProjectableBroadcastsTvs`

Reply:
changed.